transport_nantes/topicblog/views.py

```diff
@@ -372,12 +384,18 @@ def _create_email_object(self, tb_object, context: dict,
                              recipient_list: list, send_record,
                              from_email: str = settings.DEFAULT_FROM_EMAIL) \
             -> mail.EmailMultiAlternatives:
-        """
-        To send emails to multiple persons, django send_mail function isn't
-        enough, we need to create a EmailMultiAlternatives object to be able,
-        for example to put recipients in BCC field or add attachments.
-        It also improves the performances by reusing the same connexion to SMTP
-        server.
+        """Create an EmailMultiAlternatives object.
+
+        To send emails to multiple persons, django send_mail function
+        isn't enough, we need to create a EmailMultiAlternatives
+        object to be able, for example to put recipients in BCC field
+        or add attachments.  It also improves the performances by
+        reusing the same connexion to SMTP server.  **BUG 1** We don't
+        actually use an SMTP server dirctly, we are using the SES API.
+        **BUG 2** We probably don't want ever to send to a list, since
```

Comment:
The email objects are created and sent individually, we don't add everyone to a single mail so this isn't a concern I think ?

Reply:
The variable is nonetheless `recipient_list`.  So we might not use it as a list but it's still something to fix so that it isn't a list.

Reply:
We need to loop through an iterable at some point, how do you define to who you're sending it to otherwise ?

Reply:
The thing that calls create mail already loops over users.  The create mail function should simply create a send record, no?  Or maybe it's further down and it should create the mail object from the send record.  But it should surely match one-to-one with a send record (and so a single recipient).

Reply:
Ha, I see the confusion : The mail object takes a list for the recipient argument, but that list is only 1 sized : 
![image](https://user-images.githubusercontent.com/70256364/173864841-1eb6eb6f-f8c5-45e4-9cde-b4f74ac1f9e2.png)